packages/conform-valibot/tests/coercion/schema/config.test.ts

```diff
@@ -0,0 +1,18 @@
+import { config, empty, object, pipe, string } from 'valibot';
```

Comment:
Thanks for creating the test.
`fallback` and `config` are actions, not schemas, so could you put them in `tests/coercion/action/config.test.ts`?

Reply:
`fallback` and `config` are categorized as methods so I moved the test files to `tests/coercion/method/`.

By the way, since `test/coercion/method/` directory is created, it might be better to move some test cases in [`test/coercion/schema/wrap.test.ts`](https://github.com/edmundhung/conform/blob/33de60262d5cf73ed12761eed9d018d890d2e753/packages/conform-valibot/tests/coercion/schema/wrap.test.ts#L50-L86) and [`test/coercion/schema/wrapAsync.test.ts`](https://github.com/edmundhung/conform/blob/33de60262d5cf73ed12761eed9d018d890d2e753/packages/conform-valibot/tests/coercion/schema/wrapAsync.test.ts#L55-L108) into `test/coercion/method/pipe{,Async}.test.ts`.

Reply:
You're right.
I'll move on to that in the future.